iwyu_tool.py

```diff
@@ -497,7 +497,10 @@ def custom_help():
                         choices=FORMATTERS.keys(), default=DEFAULT_FORMAT,
                         help='Output format (default: %s)' % DEFAULT_FORMAT)
     parser.add_argument('-j', '--jobs', type=int, default=1,
-                        help='Number of concurrent subprocesses')
+                        nargs='?', const=0,
+                        help='Number of concurrent subprocesses. '
```

Comment:
This fits on two lines with 80-column width:
```
                        help='Number of concurrent subprocesses. If zero, will '
                             'try to match the logical cores of the system.')

```

Reply:
Oops, lost the leading parenthesis, let's do it as:
```
                        help=('Number of concurrent subprocesses. If zero, '
                              'will try to match the logical cores of the '
                              'system.')
```
to help editors that understand ()-wrapped multiline strings.

Reply:
See the `source` argument for conventional wrapping. The others seem to have snuck through review before.